python312Packages.openai: 1.44.0 -> 1.45.1

malob opened the pull request:
## Description of changes

Update to latest version. No changes that effect build.

Diff: https://github.com/openai/openai-python/compare/refs/tags/v1.44.0...v1.45.0
Changelog: https://github.com/openai/openai-python/releases/tag/v1.45.0

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

NickCao commented:
Result of `nixpkgs-review pr 341658` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages marked as broken and skipped:</summary>
  <ul>
    <li>chatgpt-retrieval-plugin</li>
    <li>chatgpt-retrieval-plugin.dist</li>
    <li>khoj</li>
    <li>khoj.dist</li>
  </ul>
</details>
<details>
  <summary>16 packages failed to build:</summary>
  <ul>
    <li>python311Packages.graphrag</li>
    <li>python311Packages.graphrag.dist</li>
    <li>python311Packages.llama-index</li>
    <li>python311Packages.llama-index-readers-file</li>
    <li>python311Packages.llama-index-readers-file.dist</li>
    <li>python311Packages.llama-index-readers-s3</li>
    <li>python311Packages.llama-index-readers-s3.dist</li>
    <li>python311Packages.llama-index.dist</li>
    <li>python311Packages.private-gpt</li>
    <li>python311Packages.private-gpt.dist</li>
    <li>python311Packages.vllm</li>
    <li>python311Packages.vllm.dist</li>
    <li>python312Packages.graphrag</li>
    <li>python312Packages.graphrag.dist</li>
    <li>python312Packages.vllm</li>
    <li>python312Packages.vllm.dist</li>
  </ul>
</details>
<details>
  <summary>174 packages built:</summary>
  <ul>
    <li>aider-chat</li>
    <li>aider-chat.dist</li>
    <li>chatblade</li>
    <li>chatblade.dist</li>
    <li>checkov</li>
    <li>checkov.dist</li>
    <li>home-assistant-component-tests.openai_conversation</li>
    <li>imaginer</li>
    <li>llm (python312Packages.llm)</li>
    <li>llm.dist (python312Packages.llm.dist)</li>
    <li>mealie</li>
    <li>mealie.dist</li>
    <li>open-interpreter (python312Packages.open-interpreter)</li>
    <li>open-interpreter.dist (python312Packages.open-interpreter.dist)</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>openai (python312Packages.openai)</li>
    <li>openai.dist (python312Packages.openai.dist)</li>
    <li>openllm</li>
    <li>openllm.dist</li>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>python311Packages.guidance</li>
    <li>python311Packages.guidance.dist</li>
    <li>python311Packages.instructor</li>
    <li>python311Packages.instructor.dist</li>
    <li>python311Packages.langchain</li>
    <li>python311Packages.langchain-aws</li>
    <li>python311Packages.langchain-aws.dist</li>
    <li>python311Packages.langchain-azure-dynamic-sessions</li>
    <li>python311Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python311Packages.langchain-chroma</li>
    <li>python311Packages.langchain-chroma.dist</li>
    <li>python311Packages.langchain-community</li>
    <li>python311Packages.langchain-community.dist</li>
    <li>python311Packages.langchain-core</li>
    <li>python311Packages.langchain-core.dist</li>
    <li>python311Packages.langchain-huggingface</li>
    <li>python311Packages.langchain-huggingface.dist</li>
    <li>python311Packages.langchain-mongodb</li>
    <li>python311Packages.langchain-mongodb.dist</li>
    <li>python311Packages.langchain-openai</li>
    <li>python311Packages.langchain-openai.dist</li>
    <li>python311Packages.langchain-standard-tests</li>
    <li>python311Packages.langchain-standard-tests.dist</li>
    <li>python311Packages.langchain-text-splitters</li>
    <li>python311Packages.langchain-text-splitters.dist</li>
    <li>python311Packages.langchain.dist</li>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python311Packages.langsmith</li>
    <li>python311Packages.langsmith.dist</li>
    <li>python311Packages.litellm</li>
    <li>python311Packages.litellm.dist</li>
    <li>python311Packages.llama-index-agent-openai</li>
    <li>python311Packages.llama-index-agent-openai.dist</li>
    <li>python311Packages.llama-index-cli</li>
    <li>python311Packages.llama-index-cli.dist</li>
    <li>python311Packages.llama-index-core</li>
    <li>python311Packages.llama-index-core.dist</li>
    <li>python311Packages.llama-index-embeddings-gemini</li>
    <li>python311Packages.llama-index-embeddings-gemini.dist</li>
    <li>python311Packages.llama-index-embeddings-google</li>
    <li>python311Packages.llama-index-embeddings-google.dist</li>
    <li>python311Packages.llama-index-embeddings-huggingface</li>
    <li>python311Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python311Packages.llama-index-embeddings-ollama</li>
    <li>python311Packages.llama-index-embeddings-ollama.dist</li>
    <li>python311Packages.llama-index-embeddings-openai</li>
    <li>python311Packages.llama-index-embeddings-openai.dist</li>
    <li>python311Packages.llama-index-graph-stores-nebula</li>
    <li>python311Packages.llama-index-graph-stores-nebula.dist</li>
    <li>python311Packages.llama-index-graph-stores-neo4j</li>
    <li>python311Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python311Packages.llama-index-graph-stores-neptune</li>
    <li>python311Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python311Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python311Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python311Packages.llama-index-legacy</li>
    <li>python311Packages.llama-index-legacy.dist</li>
    <li>python311Packages.llama-index-llms-ollama</li>
    <li>python311Packages.llama-index-llms-ollama.dist</li>
    <li>python311Packages.llama-index-llms-openai</li>
    <li>python311Packages.llama-index-llms-openai-like</li>
    <li>python311Packages.llama-index-llms-openai-like.dist</li>
    <li>python311Packages.llama-index-llms-openai.dist</li>
    <li>python311Packages.llama-index-multi-modal-llms-openai</li>
    <li>python311Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python311Packages.llama-index-program-openai</li>
    <li>python311Packages.llama-index-program-openai.dist</li>
    <li>python311Packages.llama-index-question-gen-openai</li>
    <li>python311Packages.llama-index-question-gen-openai.dist</li>
    <li>python311Packages.llama-index-readers-database</li>
    <li>python311Packages.llama-index-readers-database.dist</li>
    <li>python311Packages.llama-index-readers-json</li>
    <li>python311Packages.llama-index-readers-json.dist</li>
    <li>python311Packages.llama-index-readers-llama-parse</li>
    <li>python311Packages.llama-index-readers-llama-parse.dist</li>
    <li>python311Packages.llama-index-readers-twitter</li>
    <li>python311Packages.llama-index-readers-twitter.dist</li>
    <li>python311Packages.llama-index-readers-txtai</li>
    <li>python311Packages.llama-index-readers-txtai.dist</li>
    <li>python311Packages.llama-index-readers-weather</li>
    <li>python311Packages.llama-index-readers-weather.dist</li>
    <li>python311Packages.llama-index-vector-stores-chroma</li>
    <li>python311Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python311Packages.llama-index-vector-stores-google</li>
    <li>python311Packages.llama-index-vector-stores-google.dist</li>
    <li>python311Packages.llama-index-vector-stores-postgres</li>
    <li>python311Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python311Packages.llama-index-vector-stores-qdrant</li>
    <li>python311Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python311Packages.llama-parse</li>
    <li>python311Packages.llama-parse.dist</li>
    <li>python311Packages.llm</li>
    <li>python311Packages.llm.dist</li>
    <li>python311Packages.llmx</li>
    <li>python311Packages.llmx.dist</li>
    <li>python311Packages.open-interpreter</li>
    <li>python311Packages.open-interpreter.dist</li>
    <li>python311Packages.openai</li>
    <li>python311Packages.openai.dist</li>
    <li>python312Packages.guidance</li>
    <li>python312Packages.guidance.dist</li>
    <li>python312Packages.instructor</li>
    <li>python312Packages.instructor.dist</li>
    <li>python312Packages.langchain</li>
    <li>python312Packages.langchain-aws</li>
    <li>python312Packages.langchain-aws.dist</li>
    <li>python312Packages.langchain-azure-dynamic-sessions</li>
    <li>python312Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
    <li>python312Packages.langchain-community</li>
    <li>python312Packages.langchain-community.dist</li>
    <li>python312Packages.langchain-core</li>
    <li>python312Packages.langchain-core.dist</li>
    <li>python312Packages.langchain-huggingface</li>
    <li>python312Packages.langchain-huggingface.dist</li>
    <li>python312Packages.langchain-mongodb</li>
    <li>python312Packages.langchain-mongodb.dist</li>
    <li>python312Packages.langchain-openai</li>
    <li>python312Packages.langchain-openai.dist</li>
    <li>python312Packages.langchain-standard-tests</li>
    <li>python312Packages.langchain-standard-tests.dist</li>
    <li>python312Packages.langchain-text-splitters</li>
    <li>python312Packages.langchain-text-splitters.dist</li>
    <li>python312Packages.langchain.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph.dist</li>
    <li>python312Packages.langsmith</li>
    <li>python312Packages.langsmith.dist</li>
    <li>python312Packages.litellm</li>
    <li>python312Packages.litellm.dist</li>
    <li>python312Packages.llmx</li>
    <li>python312Packages.llmx.dist</li>
    <li>shell-genie</li>
    <li>shell-genie.dist</li>
    <li>shell-gpt</li>
    <li>shell-gpt.dist</li>
    <li>sploitscan</li>
    <li>sploitscan.dist</li>
  </ul>
</details>

malob committed:
python312Packages.openai: 1.44.0 -> 1.45.1

Diff: https://github.com/openai/openai-python/compare/refs/tags/v1.44.0...v1.45.1

Changelog: https://github.com/openai/openai-python/releases/tag/v1.45.1

malob commented:
Result of `nixpkgs-review pr 341658` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>80 packages marked as broken and skipped:</summary>
  <ul>
    <li>chatgpt-retrieval-plugin</li>
    <li>chatgpt-retrieval-plugin.dist</li>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>mealie</li>
    <li>mealie.dist</li>
    <li>open-interpreter</li>
    <li>open-interpreter.dist</li>
    <li>python311Packages.llama-index</li>
    <li>python311Packages.llama-index-agent-openai</li>
    <li>python311Packages.llama-index-agent-openai.dist</li>
    <li>python311Packages.llama-index-cli</li>
    <li>python311Packages.llama-index-cli.dist</li>
    <li>python311Packages.llama-index-core</li>
    <li>python311Packages.llama-index-core.dist</li>
    <li>python311Packages.llama-index-embeddings-gemini</li>
    <li>python311Packages.llama-index-embeddings-gemini.dist</li>
    <li>python311Packages.llama-index-embeddings-google</li>
    <li>python311Packages.llama-index-embeddings-google.dist</li>
    <li>python311Packages.llama-index-embeddings-huggingface</li>
    <li>python311Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python311Packages.llama-index-embeddings-ollama</li>
    <li>python311Packages.llama-index-embeddings-ollama.dist</li>
    <li>python311Packages.llama-index-embeddings-openai</li>
    <li>python311Packages.llama-index-embeddings-openai.dist</li>
    <li>python311Packages.llama-index-graph-stores-nebula</li>
    <li>python311Packages.llama-index-graph-stores-nebula.dist</li>
    <li>python311Packages.llama-index-graph-stores-neo4j</li>
    <li>python311Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python311Packages.llama-index-graph-stores-neptune</li>
    <li>python311Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python311Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python311Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python311Packages.llama-index-legacy</li>
    <li>python311Packages.llama-index-legacy.dist</li>
    <li>python311Packages.llama-index-llms-ollama</li>
    <li>python311Packages.llama-index-llms-ollama.dist</li>
    <li>python311Packages.llama-index-llms-openai</li>
    <li>python311Packages.llama-index-llms-openai-like</li>
    <li>python311Packages.llama-index-llms-openai-like.dist</li>
    <li>python311Packages.llama-index-llms-openai.dist</li>
    <li>python311Packages.llama-index-multi-modal-llms-openai</li>
    <li>python311Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python311Packages.llama-index-program-openai</li>
    <li>python311Packages.llama-index-program-openai.dist</li>
    <li>python311Packages.llama-index-question-gen-openai</li>
    <li>python311Packages.llama-index-question-gen-openai.dist</li>
    <li>python311Packages.llama-index-readers-database</li>
    <li>python311Packages.llama-index-readers-database.dist</li>
    <li>python311Packages.llama-index-readers-file</li>
    <li>python311Packages.llama-index-readers-file.dist</li>
    <li>python311Packages.llama-index-readers-json</li>
    <li>python311Packages.llama-index-readers-json.dist</li>
    <li>python311Packages.llama-index-readers-llama-parse</li>
    <li>python311Packages.llama-index-readers-llama-parse.dist</li>
    <li>python311Packages.llama-index-readers-s3</li>
    <li>python311Packages.llama-index-readers-s3.dist</li>
    <li>python311Packages.llama-index-readers-twitter</li>
    <li>python311Packages.llama-index-readers-twitter.dist</li>
    <li>python311Packages.llama-index-readers-txtai</li>
    <li>python311Packages.llama-index-readers-txtai.dist</li>
    <li>python311Packages.llama-index-readers-weather</li>
    <li>python311Packages.llama-index-readers-weather.dist</li>
    <li>python311Packages.llama-index-vector-stores-chroma</li>
    <li>python311Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python311Packages.llama-index-vector-stores-google</li>
    <li>python311Packages.llama-index-vector-stores-google.dist</li>
    <li>python311Packages.llama-index-vector-stores-postgres</li>
    <li>python311Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python311Packages.llama-index-vector-stores-qdrant</li>
    <li>python311Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python311Packages.llama-index.dist</li>
    <li>python311Packages.llama-parse</li>
    <li>python311Packages.llama-parse.dist</li>
    <li>python311Packages.open-interpreter</li>
    <li>python311Packages.open-interpreter.dist</li>
    <li>python311Packages.private-gpt</li>
    <li>python311Packages.private-gpt.dist</li>
    <li>python312Packages.open-interpreter</li>
    <li>python312Packages.open-interpreter.dist</li>
  </ul>
</details>
<details>
  <summary>109 packages built:</summary>
  <ul>
    <li>aider-chat</li>
    <li>aider-chat.dist</li>
    <li>chatblade</li>
    <li>chatblade.dist</li>
    <li>checkov</li>
    <li>checkov.dist</li>
    <li>imaginer</li>
    <li>llm (python312Packages.llm)</li>
    <li>llm.dist (python312Packages.llm.dist)</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>openai (python312Packages.openai)</li>
    <li>openai.dist (python312Packages.openai.dist)</li>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>python311Packages.graphrag</li>
    <li>python311Packages.graphrag.dist</li>
    <li>python311Packages.guidance</li>
    <li>python311Packages.guidance.dist</li>
    <li>python311Packages.instructor</li>
    <li>python311Packages.instructor.dist</li>
    <li>python311Packages.langchain</li>
    <li>python311Packages.langchain-aws</li>
    <li>python311Packages.langchain-aws.dist</li>
    <li>python311Packages.langchain-azure-dynamic-sessions</li>
    <li>python311Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python311Packages.langchain-chroma</li>
    <li>python311Packages.langchain-chroma.dist</li>
    <li>python311Packages.langchain-community</li>
    <li>python311Packages.langchain-community.dist</li>
    <li>python311Packages.langchain-core</li>
    <li>python311Packages.langchain-core.dist</li>
    <li>python311Packages.langchain-huggingface</li>
    <li>python311Packages.langchain-huggingface.dist</li>
    <li>python311Packages.langchain-mongodb</li>
    <li>python311Packages.langchain-mongodb.dist</li>
    <li>python311Packages.langchain-openai</li>
    <li>python311Packages.langchain-openai.dist</li>
    <li>python311Packages.langchain-standard-tests</li>
    <li>python311Packages.langchain-standard-tests.dist</li>
    <li>python311Packages.langchain-text-splitters</li>
    <li>python311Packages.langchain-text-splitters.dist</li>
    <li>python311Packages.langchain.dist</li>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python311Packages.langsmith</li>
    <li>python311Packages.langsmith.dist</li>
    <li>python311Packages.litellm</li>
    <li>python311Packages.litellm.dist</li>
    <li>python311Packages.llm</li>
    <li>python311Packages.llm.dist</li>
    <li>python311Packages.llmx</li>
    <li>python311Packages.llmx.dist</li>
    <li>python311Packages.openai</li>
    <li>python311Packages.openai.dist</li>
    <li>python312Packages.graphrag</li>
    <li>python312Packages.graphrag.dist</li>
    <li>python312Packages.guidance</li>
    <li>python312Packages.guidance.dist</li>
    <li>python312Packages.instructor</li>
    <li>python312Packages.instructor.dist</li>
    <li>python312Packages.langchain</li>
    <li>python312Packages.langchain-aws</li>
    <li>python312Packages.langchain-aws.dist</li>
    <li>python312Packages.langchain-azure-dynamic-sessions</li>
    <li>python312Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
    <li>python312Packages.langchain-community</li>
    <li>python312Packages.langchain-community.dist</li>
    <li>python312Packages.langchain-core</li>
    <li>python312Packages.langchain-core.dist</li>
    <li>python312Packages.langchain-huggingface</li>
    <li>python312Packages.langchain-huggingface.dist</li>
    <li>python312Packages.langchain-mongodb</li>
    <li>python312Packages.langchain-mongodb.dist</li>
    <li>python312Packages.langchain-openai</li>
    <li>python312Packages.langchain-openai.dist</li>
    <li>python312Packages.langchain-standard-tests</li>
    <li>python312Packages.langchain-standard-tests.dist</li>
    <li>python312Packages.langchain-text-splitters</li>
    <li>python312Packages.langchain-text-splitters.dist</li>
    <li>python312Packages.langchain.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph.dist</li>
    <li>python312Packages.langsmith</li>
    <li>python312Packages.langsmith.dist</li>
    <li>python312Packages.litellm</li>
    <li>python312Packages.litellm.dist</li>
    <li>python312Packages.llmx</li>
    <li>python312Packages.llmx.dist</li>
    <li>shell-genie</li>
    <li>shell-genie.dist</li>
    <li>shell-gpt</li>
    <li>shell-gpt.dist</li>
    <li>sploitscan</li>
    <li>sploitscan.dist</li>
  </ul>
</details>

natsukium commented:
Result of `nixpkgs-review pr 341658` at 8cee44ca run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages marked as broken and skipped:</summary>
  <ul>
    <li>chatgpt-retrieval-plugin</li>
    <li>khoj</li>
    <li>python311Packages.llama-index</li>
    <li>python311Packages.llama-index-readers-file</li>
    <li>python311Packages.llama-index-readers-s3</li>
    <li>python311Packages.private-gpt</li>
  </ul>
</details>
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/natsukium/6ee09a92c20a42f4ac7cdf2464d887ca">python311Packages.vllm</a></li>
    <li><a href="https://gist.github.com/natsukium/58827b582a510bfc83f4dcf12c7d0fa4">python312Packages.vllm</a></li>
  </ul>
</details>
<details>
  <summary>90 packages built successfully:</summary>
  <ul>
    <li>aider-chat</li>
    <li>chatblade</li>
    <li>checkov</li>
    <li>home-assistant-component-tests.openai_conversation</li>
    <li>imaginer</li>
    <li>llm (python312Packages.llm)</li>
    <li>mealie</li>
    <li>open-interpreter (python312Packages.open-interpreter)</li>
    <li>open-webui</li>
    <li>openai (python312Packages.openai)</li>
    <li>openllm</li>
    <li>pentestgpt</li>
    <li>python311Packages.graphrag</li>
    <li>python311Packages.guidance</li>
    <li>python311Packages.instructor</li>
    <li>python311Packages.langchain</li>
    <li>python311Packages.langchain-aws</li>
    <li>python311Packages.langchain-azure-dynamic-sessions</li>
    <li>python311Packages.langchain-chroma</li>
    <li>python311Packages.langchain-community</li>
    <li>python311Packages.langchain-core</li>
    <li>python311Packages.langchain-huggingface</li>
    <li>python311Packages.langchain-mongodb</li>
    <li>python311Packages.langchain-openai</li>
    <li>python311Packages.langchain-standard-tests</li>
    <li>python311Packages.langchain-text-splitters</li>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langsmith</li>
    <li>python311Packages.litellm</li>
    <li>python311Packages.llama-index-agent-openai</li>
    <li>python311Packages.llama-index-cli</li>
    <li>python311Packages.llama-index-core</li>
    <li>python311Packages.llama-index-embeddings-gemini</li>
    <li>python311Packages.llama-index-embeddings-google</li>
    <li>python311Packages.llama-index-embeddings-huggingface</li>
    <li>python311Packages.llama-index-embeddings-ollama</li>
    <li>python311Packages.llama-index-embeddings-openai</li>
    <li>python311Packages.llama-index-graph-stores-nebula</li>
    <li>python311Packages.llama-index-graph-stores-neo4j</li>
    <li>python311Packages.llama-index-graph-stores-neptune</li>
    <li>python311Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python311Packages.llama-index-legacy</li>
    <li>python311Packages.llama-index-llms-ollama</li>
    <li>python311Packages.llama-index-llms-openai</li>
    <li>python311Packages.llama-index-llms-openai-like</li>
    <li>python311Packages.llama-index-multi-modal-llms-openai</li>
    <li>python311Packages.llama-index-program-openai</li>
    <li>python311Packages.llama-index-question-gen-openai</li>
    <li>python311Packages.llama-index-readers-database</li>
    <li>python311Packages.llama-index-readers-json</li>
    <li>python311Packages.llama-index-readers-llama-parse</li>
    <li>python311Packages.llama-index-readers-twitter</li>
    <li>python311Packages.llama-index-readers-txtai</li>
    <li>python311Packages.llama-index-readers-weather</li>
    <li>python311Packages.llama-index-vector-stores-chroma</li>
    <li>python311Packages.llama-index-vector-stores-google</li>
    <li>python311Packages.llama-index-vector-stores-postgres</li>
    <li>python311Packages.llama-index-vector-stores-qdrant</li>
    <li>python311Packages.llama-parse</li>
    <li>python311Packages.llm</li>
    <li>python311Packages.llmx</li>
    <li>python311Packages.open-interpreter</li>
    <li>python311Packages.openai</li>
    <li>python312Packages.graphrag</li>
    <li>python312Packages.guidance</li>
    <li>python312Packages.instructor</li>
    <li>python312Packages.langchain</li>
    <li>python312Packages.langchain-aws</li>
    <li>python312Packages.langchain-azure-dynamic-sessions</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-community</li>
    <li>python312Packages.langchain-core</li>
    <li>python312Packages.langchain-huggingface</li>
    <li>python312Packages.langchain-mongodb</li>
    <li>python312Packages.langchain-openai</li>
    <li>python312Packages.langchain-standard-tests</li>
    <li>python312Packages.langchain-text-splitters</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langsmith</li>
    <li>python312Packages.litellm</li>
    <li>python312Packages.llmx</li>
    <li>shell-genie</li>
    <li>shell-gpt</li>
    <li>sploitscan</li>
  </ul>
</details>

natsukium reviewed (approved):
LGTM
